### ISSUE TITLE
UndeclaredThrowableException when trying to update plugins

### ISSUE THREAD

**Issue by msoutopico** [chelobaka/omegat_scripts#2]
### Preconditions

OmegaT 4.3.2 installed in a machine running Windows 10.
Script `application_startup\updateConfigBundle.groovy` is version 0.5.1, and `def customUrl = ""`
File `customisation.properties` has url: `https://cat.capstan.be/OmegaT/dev2021/index.php`
Local customization version `57_cs0 (2020-11-25)`
Remote customization version `58_c0p (2020-11-26)`

Local plugins:
* okapiFiltersForOmegaT-1.6-m40-capstan.jar
* omegat-bidimarkers-0.2.0-all.jar
* plugin-omt-package-1.6.3.jar

Remote plugins:
* okapiFiltersForOmegaT-1.8-1.40.0-capstan.jar (NEW)
* omegat-bidimarkers-0.2.0-all.jar
* plugin-omt-package-1.6.3.jar

### Steps to reproduce 

Just launch OmegaT. 

### Expected results

The script runs and sees that `okapiFiltersForOmegaT-1.8-1.40.0-capstan.jar` is a new version of `okapiFiltersForOmegaT-1.6-m40-capstan.jar`, so the former must replace the latter. 

The new version is downloaded and, after OmegaT shuts down, the old version of the plugin is deleted (by means of the bat file triggered by the script).

### Actual results

The script throws error `javax.script.ScriptException: javax.script.ScriptException: java.lang.reflect.UndeclaredThrowableException` in the console and the progress bar runs forever.

See the full output:
```
========================================
          Customisation Update
========================================
Local customisation version is 56_cs0.
Remote customisation version is 57_csp.
Customisation update available:
* Config needs to be updated.
* Scripts need to be updated.
* Plugins need to be updated.
----------------------------------------
config.zip is being downloaded...
config.zip is being unpacked...
Customisation is being updated...
OmegaT will need to be restarted.
    -- Done --
----------------------------------------
scripts.zip is being downloaded...
scripts.zip is being unpacked...
Scripts are being installed...
Scripts provided in the customisation bundle copied to C:\Users\souto\AppData\Roaming\OmegaT\scripts.
    -- Done --
----------------------------------------
plugins.zip is being downloaded...
plugins.zip is being unpacked...
Plugins are being installed...
An error occurred
javax.script.ScriptException: javax.script.ScriptException: java.lang.reflect.UndeclaredThrowableException
```

See ![screencast](https://recordit.co/NjODOQiWz0)

**Comment by msoutopico** [comment]
[OmegaT.log](https://github.com/chelobaka/omegat_scripts/files/5609361/OmegaT.log)
[customisation_202011272031.log](https://github.com/chelobaka/omegat_scripts/files/5609362/customisation_202011272031.log)

**Comment by chelobaka** [comment]
Fixed in 0.5.2.
